odpf/optimus/runtime_service.proto

```diff
@@ -680,10 +680,13 @@ message ReplayRequest {
     string start_date = 4;
     string end_date = 5;
     bool force = 6;
+    bool ignore_downstream = 7;
+    string allowed_downstream = 8;
```

Comment:
is it the namespace name? Can it only be one? Is it a delimited string?

Reply:
for now, the possible value is either: '*' or empty. but in the future if we decided to allow replaying downstream for some certain namespace (not all), we can put it a delimited string. wdyt?

Reply:
if `allowed_downstream` is empty, isn't it the same as `ignore_downstream`?

Reply:
as discussed, updated to deprecate ignore_downstream and make allowed_downstream as repeated string.